actionpack/lib/action_controller/caching.rb

```diff
@@ -23,65 +22,24 @@ module ActionController
   #   config.action_controller.cache_store = :mem_cache_store, Memcached::Rails.new('localhost:11211')
   #   config.action_controller.cache_store = MyOwnStore.new('parameter')
   module Caching
-    extend ActiveSupport::Concern
     extend ActiveSupport::Autoload
```

Comment:
Remove this line will cause `undefined method eager_load!` error in [here](https://github.com/rails/rails/blob/master/actionpack%2Flib%2Faction_controller.rb#L49).